sidebars.js

```diff
@@ -50,11 +51,6 @@ const sidebars = {
               type: 'autogenerated',
               dirName: 'sdks/javascript-sdk'
             },
-            {
-              type: 'link',
-              label: 'Documentation',
-              href: 'https://gobeyondidentity.github.io/bi-sdk-js/',
-            },
```

Comment:
It seems like I was wrong and we might be keeping the individual folders after all. Jen's making individual reference documents. So it might make sense to leave these here. I think it's fine to keep them linked in the set up overview too.

Reply:
Ok, I didn't think we were removing the individual folders yet anyway. I can put these back.